### ISSUE TITLE
Error: Invalid resource type: The provider hashicorp/awscc does not support resource type "awscc_datazone_environment".

### ISSUE THREAD

**Issue by molivo123** [hashicorp/terraform-provider-awscc#1563]
### Community Note

    Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
    Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
    If you are interested in working on this issue or have submitted a pull request, please leave a comment
    The resources and data sources in this provider are generated from the CloudFormation schema, so they can only support the actions that the underlying schema supports. For this reason submitted bugs should be limited to defects in the generation and runtime code of the provider. Customizing behavior of the resource, or noting a gap in behavior are not valid bugs and should be submitted as enhancements to AWS via the CloudFormation Open Coverage Roadmap.

### Terraform CLI and Terraform AWS Cloud Control Provider Version

Terraform v1.0.7
on linux_amd64

    provider registry.terraform.io/hashicorp/aws v5.43.0
    provider registry.terraform.io/hashicorp/awscc v0.72.1

### Affected Resource(s)

AWSCC and Amazon Datazone

environment.tf

```
resource "awscc_datazone_environment" "example_environment" {
    domain_identifier = awscc_datazone_domain.this.id
    environment_profile_identifier = awscc_datazone_environment_profile.example_profile.id
    name = "company_env"
    project_identifier = awscc_datazone_project.engineering_test_project.id
}
```

### Debugging Output
```
Error: Invalid resource type
│ 
│   on environment.tf line 1, in resource "awscc_datazone_environment" "example_environment":
│    1: resource "awscc_datazone_environment" "example_environment" {
│ 
│ The provider hashicorp/awscc does not support resource type "awscc_datazone_environment".
```

### Expected Behavior

I should be able to create an Amazon Datazone environment

### Actual Beavior

Error out saying awscc does not support that resource type

### Steps to Reproduce

    terraform plan

Important Factoids

N/A

### References
https://registry.terraform.io/providers/hashicorp/awscc/latest/docs/resources/datazone_environment

**Comment by quixoticmonk** [comment]
`awscc_datazone_environment` was included from [v0.73.0](https://github.com/hashicorp/terraform-provider-awscc/releases/tag/v0.73.0). Are you able to upgrade to 0.73 or later ?

**Comment by molivo123** [comment]
@quixoticmonk yes I can and that fixed my issue, thank you!
